Feature: Update intro to Chingu and add Manifesto (#176)
chingu-voyages/Handbook: README.md

## What is Chingu❓

Chingu is an organization whose mission is to help Web Development professionals
(Developers, UI/UX Designers, Scrum Product Owners, and Scrum Masters)  
transform what they’ve learned in university studies, private courses, videos, and 

> individual studies into the experience needed to get jobs. 🎬

Reviewer: ```suggestion
individual studies into the experience needed to launch their careers. 🎬
```

## How did Chingu get started?

The idea for Chingu came from one of our original co-founders who had spent 
thousands on a top-flight bootcamp and excelled in the program of study. 

> But, upon completing the course of study he found he knew the technology parts of 

Reviewer: ```suggestion
But, upon completing the course of study, he found he knew the technology parts of 
```

Web Development, but didn’t have the skills expected from a Junior Developer 
including how to design applications, work in a team, use distributed tools like 
git & GitHub in a team, and use Agile frameworks like Scrum.

Despite having done well at this bootcamp and having invested considerable time and 

> money into it his employment prospects were dim. This raised the question that 

Reviewer: ```suggestion
money into it, his employment prospects were dim. This raised the question that 
```

> started Chingu - “How can I gain the experience to become employable?”. As a result of this Chingu was started in 2016.

Reviewer: ```suggestion
started Chingu - “How can I gain the experience to become employable?” As a result of this, Chingu was started in 2016.
```

> started Chingu - “How can I gain the experience to become employable?” As a result of this Chingu was started in 2016.

Reviewer: ```suggestion
started Chingu - “How can I gain the experience to become employable?” As a result of this, Chingu was started in 2016.
```

Our programs and services are geared to help Web Developers, UI/UX Designers, and
Agile Product Owners & Scrum Masters.

## What are our core principles?

Given our mission, what are some of the high-level principles that govern what we 

> do and how we do it? Click [here](./docs/manifesto.md) to read more about what is important to us in

Author: Resolved
